homework-1/js/script.js

```diff
@@ -0,0 +1,136 @@
+////////////menu///////////
+const iconMenu = document.querySelector('.menu__icon');
+  if(iconMenu) {
+    const menuBody = document.querySelector('.menu__body');
+    iconMenu.addEventListener("click", function(e){
+      document.body.classList.toggle('lock');
+      iconMenu.classList.toggle('active');
+      menuBody.classList.toggle('active');
+    });
+  }
+
+  window.addEventListener('scroll', () => {
+    let scrollDistance = window.scrollY;
+      document.querySelectorAll('.section').forEach((el, i) => {
+        if (el.offsetTop - document.querySelector('#banner-submenu').clientHeight <= scrollDistance) {
+          document.querySelectorAll('.submenu a').forEach((el) => {
+            if (el.classList.contains('tips-active')) {
+              el.classList.remove('tips-active');
+            }
+          });
+  
+          document.querySelectorAll('.submenu li')[i].querySelector('a').classList.add('tips-active');
+        }
+      });
+  });
+///////////accordeon///////////////
+
+const tipLink = document.querySelectorAll('.tips__link');
```

Comment:
I suppose here not the pnly one node element. Please rename varible such tipLinks

Reply:
fixed a bug

Reply:
done

Reply:
done